app/views/time_based_price_policies/_price_policy_fields.html.haml

```diff
@@ -43,7 +46,6 @@
           %td
         - if local_assigns[:cancellation]
           %td
-      - if @product.duration_pricing_mode?
```

Comment:
Why did this get removed?

Reply:
good catch! this was an error

Reply:
Oh I remember now, it's not an error. There are two ifs with the same condition one after another so I removed the second one